Gem/Transfer/Types/PriceImpact.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+import Foundation
+import Style
+import SwiftUICore
+
+enum PriceImpactType: Equatable {
+    case none
+    case low(String)
+    case medium(String)
+    case high(String)
+    case positive(String)
```

Comment:
```suggestion
enum PriceImpactType: Equatable {
    case none
    case low
    case medium
    case high
    case positive
```

Keep `PriceImpactType` simple, it should just describe the type and not the value(especially in string)